dash/_cli.py

```diff
@@ -0,0 +1,171 @@
+import argparse
+import importlib
+import sys
+from typing import Any, Dict
+
+from dash import Dash
+
+
+def load_app(app_path: str) -> Dash:
+    """
+    Load a Dash app instance from a string like "module:variable".
+
+    :param app_path: The import path to the Dash app instance.
+    :return: The loaded Dash app instance.
+    """
+    app_split = app_path.split(":")
+    module_str = app_split[0]
+
+    if not module_str:
+        raise ValueError(f"Invalid app path: '{app_path}'. ")
+
+    try:
+        module = importlib.import_module(module_str)
+    except ImportError as e:
+        raise ImportError(f"Could not import module '{module_str}'.") from e
+
+    if len(app_split) == 2:
+        app_str = app_split[1]
+        try:
+            app_instance = getattr(module, app_str)
+        except AttributeError as e:
+            raise AttributeError(
+                f"Could not find variable '{app_str}' in module '{module_str}'."
+            ) from e
+    else:
+        for module_var in vars(module).values():
+            if isinstance(module_var, Dash):
+                app_instance = module_var
+                break
+
+    if not isinstance(app_instance, Dash):
```

Comment:
if I run with just the module name, but the module doesn't define a `Dash`, I think `app_instance` will be uninitialized at the point of the `isinstance()` check.